src/Jank/PlayerSelection/PlayerCard.tsx

```diff
@@ -0,0 +1,25 @@
+import { useParams } from 'react-router-dom';
+import defaultPicture from '../../UserManagement/user_default.png';
+import { Picture, Card } from './ui';
+
+type PlayerProps = {
+  playerId: string;
+  name: string;
+  picture: string;
+}
+
+interface ScorepadParams {
+  scorepadId: string
+}
+const PlayerCard = ({ playerId, name, picture = defaultPicture }: PlayerProps) => {
+  const { scorepadId } = useParams<ScorepadParams>();
```

Comment:
I put it together with the PlayerProbs into a CardProbs because I was not able to make 'passing multiple type props' work.

Reply:
OK sure, but what I meant was to just have the `scorepadId` as part of the `PlayerProps` instead, and simply pass it down from the parent, instead of fetching it from the route params with `useParams()` here again.

I think it'd be preferable if only the parent component (which is the root of the corresponding route) depended on the actual route. So it would be something like
```
const PlayerSelection = () => {
  const { scorepadId } = useParams<ScorepadParams>(); // access params from route here
  ...
  <PlayerCard scorepadId={scorepadId} ... /> // but use the prop explicitly here
};
```
so that `PlayerCard` doesn't have dependencies on the route. This way any changes to the route do not affect multiple components, but only the root of this specific route.

Reply:
yeah that is basically what i did i guess. I'll push it later, then you can check if we meant the same thing :D